src/web_app_skeleton/script/setup.ecr

```diff
@@ -10,6 +10,23 @@ indent() {
   done
 }
 
+# Ensure postgres is installed, and booted
+check_postgres() {
+  if ! command -v createdb > /dev/null; then
+    printf 'Please install the postgres CLI tools, then try again.\n'
+    printf 'See https://www.postgresql.org/docs/current/tutorial-install.html for install instructions.\n'
+    if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Let's put this above the other  once since if it matches it is more likely to be the one people want. A ton of people use Postgres.app and if they don't then they can check out the other instructions.